demos/blog/README.rst

```diff
@@ -1,26 +1,32 @@
 Setup
 =====
 
-Terminal 1: Run db server::
+Clone the repo, create virtualenv if necessary.
 
-    $ docker run --rm -it -p 5433:5432 postgres:10
+Operate from base folder [../aiohttp-demos/demos/blog].
```

Comment:
Is square brackets some kind of special syntax in rst?

Reply:
no